### ISSUE TITLE
Not a JSON Object: "<html>"

### ISSUE THREAD

**Issue by aspectratio1** [Headline/CleverBotAPI-Java#6]
Whenever I try the "Test" Script, I receive an error
```
Exception in thread "main" java.lang.IllegalStateException: Not a JSON Object: "<html>"
at com.google.gson.JsonElement.getAsJsonObject(JsonElement.java:91)
at com.michaelwflaherty.cleverbotapi.CleverBotQuery.sendRequest(CleverBotQuery.java:168)
at com.company.Main.main(Main.java:33)
```

I've tried with different version of gson, and no API requests are being shown on cleverbot.

**Comment by ElectronicWizard** [comment]
Getting this exception as well, just decided to try this today.

**Comment by Headline** [comment]
I've found the issue. It appears they no longer support standard `http` calls, but instead *require* `https`. I've fixed it, however I am having some issues deploying to maven central (lost my old key, have to wait for keyservers to synchronize).

1.0.3 should be out in no time. Will close once it's confirmed working.

**Comment by Headline** [comment]
All seems to check out, just waiting on maven servers to recognize my release. I'll go ahead and close this. If you have any further issues, feel free to reopen or create a new issue.

:beers:
